modules/packages/LinearAlgebra.chpl

```diff
@@ -340,64 +342,72 @@ class ExpmPadeHelper {
     return this.exactD10;
   }
 
-  proc comp_looseD4(){
-    if this.useExactOneNorm then{
+  proc comp_looseD4() {
+    if this.useExactOneNorm || isComplexType(this.eltType) then {
```

Comment:
Can't we set `useExactOneNorm` in the initializer to account for complex values and avoid this repeated `||`s?